app/assets/javascripts/form.js

```diff
@@ -0,0 +1,12 @@
+//$( ".new-box" ).click(function() {
+//  $(".form").slideDown();
+//});
+//$(function() {
+//   $(".new-box").click(function(event) {
+//     event.preventDefault();
+     //var userForm = newUserForm(".form");
+//     var userForm = newUserForm()
+     //var commentCreator = newCommentCreator(commentForm.attributes().description, commentForm.attributes().review, commentForm.attributes().comment_id, "div#comments");
+//     commentCreator.create();
+//  });
+//});
```

Comment:
Committing commented code is a bad practice. Typically, you want to keep the things you are working on in Git Branches, so that non-functional code does not pollute the codebase.